core/output/interactive.go

```diff
@@ -21,18 +21,24 @@ func NewInteractiveShell(out, errOut io.Writer, verbosity int) Output {
 		out:       out,
 		errOut:    newSpinner(errOut),
 		verbosity: verbosity,
+		level:     0,
 	}
 }
 
 type interactiveShellOutput struct {
-	out       io.Writer
-	errOut    *spinner
-	verbosity int
-	status    string
-	lock      sync.Mutex
+	out           io.Writer
+	errOut        *spinner
+	verbosity     int
+	level         int
```

Comment:
Why are we introducing new `level` variable? Couldn't we use `verbosity` here?

Reply:
Verbosity is something different: it specifies how verbose the output is (maximum level that will be printed to output)

Reply:
So `verbosity` represents the maximum verbosity that will print something and `level` is the current verbosity value?

Reply:
I see the new comments, thanks!

Reply:
Yes, or to be more precise: `verbosity` is the output's (=logger's) verbosity. `level` is the verbosity of the messages you want to output.